tests/test_library.py

```diff
@@ -260,3 +260,6 @@ def test_crazy_search(plex, movie):
     ), "Unable to search movie by year."
     assert movie not in movies.search(year=2007), "Unable to filter movie by year."
     assert movie in movies.search(actor=movie.actors[0].tag)
+    assert len(movies.search(container_start=2, maxresults=1)) == 1
```

Comment:
Maybe add a test for the case mentioned above witt
```
assert len(movies.search(container_start=2, container_size=1)) == 2
```

Reply:
Yep, at att test for container_size higher then total items, atm this will be a endless loop.